tests/org.jboss.tools.openshift.egit.test/.project

```diff
@@ -20,8 +20,14 @@
 			<arguments>
 			</arguments>
 		</buildCommand>
+		<buildCommand>
```

Comment:
please remove this, this forces the maven nature and devs to have m2e installed and participating in the build.